docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -1308,6 +1308,10 @@ GiB).
 
 If true, runs the node as a [plugin.](https://github.com/hashicorp/go-plugin) Defaults to `false`.
 
+#### `--plugin-dir` (string)
+
+Sets the directory for [plugins](https://github.com/hashicorp/go-plugin). The default value is `[DATADIR]/plugins`.
```

Comment:
AvalancheGo does not use `hashicorp/go-plugin` for plugins anymore. Can you rename `### Plugin Mode` to `### Plugins` and then reword `Sets the directory for [plugins](https://github.com/hashicorp/go-plugin)` to something like `Sets the directory for [VM plugins](../../subnets/introduction-to-vm.md)`.

Reply:
fixed  bd4506785f53e2afec17e4bb4f63ee3ef87b66a9